main.py

```diff
@@ -185,3 +188,39 @@ def get_cookie(dpid):
         """Get 8-byte integer cookie."""
         int_dpid = int(dpid.replace(":", ""), 16)
         return (0x00FFFFFFFFFFFFFF & int_dpid) | (settings.COOKIE_PREFIX << 56)
+
+    def set_flow_table_group_owner(self,
+                                   flow: dict,
+                                   group: str = "base") -> dict:
+        """Set owner, table_group and table_id
+        coloring is only allowing 'base' for now"""
+        try:
+            flow["table_id"] = self.table_group[group]
```

Comment:
No need to try catch `KeyError` here you can let it crash without the log since `log.error` is already being handled on `on_table_enabled`, so `self.table_group` should always be safe.

Reply:
Oh, got it. Changed.